app/controllers/car_models_controller.rb

```diff
@@ -0,0 +1,30 @@
+class CarModelsController < ApplicationController
+
+  def new
+    @manufacturer = Manufacturer.find(params[:manufacturer_id])
+    @car_model = CarModel.new
+  end
+
+  def create
+    @manufacturer = Manufacturer.find(params[:manufacturer_id])
+
+    if @car_model = @manufacturer.car_models.create(car_model_params)
+      redirect_to manufacturer_car_model_path(@manufacturer, @car_model)
```

Comment:
redirect_to manufacturer_car_model_path(@manufacturer, @car_model), notice: "Model created successfully"
# that notice will be available as flash[:notice] in the next action/template